### ISSUE TITLE
Results Screen Top Only bug

### ISSUE THREAD

**Issue by Scatlink** [vitor251093/KHMelonMix#152]
**Describe the bug**
In V0.3.0 if you finish or quit a mission with no items picked up or earned, the blank surrounding screen goes pink and focuses on the top screen only. This displayed correctly on V0.2.2. 

**To Reproduce**
1. Go to any mission 
2. Click on 'withdraw' to quit the mission to the Results screen
3. If no items load on the bottom screen, the Top Screen goes into Full Screen 4:3 focus with a pink border instead of the usual grey
4. This causes trying to leave the Results Screen difficult as you cannot see the bottom screen selection menu

**Expected behavior**
As stated, in V0.2.2 this wasn't a problem; both top and bottom screens on the Results Screens would display side by side with the grey screen border; regardless if any items were collected or not.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Melon Mix Version 0.3.0

**Additional context**
n/a

**Comment by Ssheph95** [comment]
I'd like to add that this also seems to occur in pretty much every mission when playing mission mode, regardless of if items were collected or not.

**Comment by Ssheph95** [comment]
Explored further with a quick new game through the tutorial levels, and I seem to have formed a more specific hypothesis. The bug seems to occur any time there are less than 4 TOTAL entries between the two columns.

![image](https://github.com/vitor251093/KHDays_FM/assets/97009666/15062e83-53aa-4d50-91a0-dc0fee24836d)
![image](https://github.com/vitor251093/KHDays_FM/assets/97009666/0f4707b2-f9ce-46a9-98b7-d453060b887f)

This seems to be consistent between different missions, rewards, and types of items. All these screencaptures were taken in "Even" screen mode, but the bug is clearly visible on the right (top) screen when and where it occurs.

**Comment by vitor251093** [comment]
@Ssheph95 are you capable of reproducing the bug with the latest Actions build?

**Comment by Ssheph95** [comment]
@vitor251093 unfortunately I'm away on work without my computer for the next few days (writing from phone). I'll check when I get back, but probably won't be able to test until Sunday or Monday, so if there's a more readily available player, you might be able to consult them faster.

**Comment by Ssheph95** [comment]
https://github.com/vitor251093/KHMelonMix/assets/97009666/b97b0b06-4888-4a31-905e-01b96e75ec3f

https://github.com/vitor251093/KHMelonMix/assets/97009666/15fe855a-c48e-45b1-a1ae-842a0fca5f73

![image](https://github.com/vitor251093/KHMelonMix/assets/97009666/68526e41-288e-438c-b9db-a6986bf7aece)


I ran a few tests and can confirm that the original issue of "top screen only" is fixed for any amount of items. However, there is still a persistent graphical oddity when only one item is present, as shown by the videos linked. This graphics thing appears only with one item, as two, three, and further don't have any issue whatsoever. The main bug that is subject of this issue has been solved though.
